consensus/bor/rest.go

```diff
@@ -25,18 +25,13 @@ type IHeimdallClient interface {
 }
 
 type HeimdallClient struct {
-	u      *url.URL
-	client http.Client
+	urlString string
```

Comment:
We dont need to mention data type with variable name here I think

Reply:
Added it because there is `url` package imported. So used `urlString` to have a meaningful name without conflicting with the imported package.